crates/merod/src/cli/config.rs

```diff
@@ -1,24 +1,44 @@
 #![allow(unused_results, reason = "Occurs in macro")]
 
-use std::env::temp_dir;
+use std::io::Write;
 use std::str::FromStr;
 
 use calimero_config::{ConfigFile, CONFIG_FILE};
 use camino::Utf8PathBuf;
-use clap::Parser;
+use clap::{Parser, ValueEnum};
+use comfy_table::{Cell, Color as ComfyColor, Table};
 use eyre::{bail, eyre, Result as EyreResult};
+use lazy_static::lazy_static;
+use similar::{ChangeTag, TextDiff};
+use termcolor::{Color as TermColor, ColorChoice, ColorSpec, StandardStream, WriteColor};
 use tokio::fs::{read_to_string, write};
 use toml_edit::{Item, Value};
 use tracing::info;
 
 use crate::cli;
 
 /// Configure the node
-#[derive(Debug, Parser)]
+#[derive(Debug, Parser, Clone)]
 pub struct ConfigCommand {
     /// Key-value pairs to be added or updated in the TOML file
     #[clap(value_name = "ARGS")]
     args: Vec<KeyValuePair>,
+
+    /// Output format for printing config
+    #[clap(long, value_name = "FORMAT", default_value = "default")]
+    print: PrintFormat,
+
+    /// Save modifications to config file
+    #[clap(short, long)]
+    save: bool,
+}
+
+#[derive(Clone, Debug, ValueEnum)]
+enum PrintFormat {
+    Default,
+    Toml,
+    Json,
+    Human,
```

Comment:
we don't need an explicit human format, since not all cases have one

when viewing, `Default` is interpreted to mean "toml"
when editing, `Default` is interpreted to mean "show diff"
when hinting, `Default` is interpreted to mean "human readable"